models/topics.models.js

```diff
@@ -0,0 +1,8 @@
+const db = require("../db/connection");
+const format = require("pg-format");
```

Comment:
You have required in pg-format here when you don't need it yet in your model. Please remove before merging to main.